Sources/TensorFlow/Core/Threading.swift

```diff
@@ -16,10 +16,51 @@
 import Darwin
 #elseif os(Windows)
 import ucrt
+import WinSDK
 #else
 import Glibc
 #endif
 
+struct ThreadLocalStorage {
+#if os(Windows)
+    typealias Key = DWORD
+#else
+    typealias Key = pthread_key_t
+#endif
+
+#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
+    typealias KeyDestructor = @convention(c) (UnsafeMutableRawPointer) -> Void
+#else
+    typealias KeyDestructor = @convention(c) (UnsafeMutableRawPointer?) -> Void
+#endif
+
+    public static func KeyCreate(_ destructor: KeyDestructor?) -> Key {
```

Comment:
[Swift API design guidelines](https://swift.org/documentation/api-design-guidelines/#begin-factory-name-with-make):
> Begin names of factory methods with "make", e.g. `x.makeIterator()`.

Please also add the `destructor:` argument label. Callers passing trailing closures should be unaffected.

```suggestion
    public static func makeKey(destructor: KeyDestructor?) -> Key {
```

Reply:
I think that I would rather make `Key` a proper abstraction.  I was thinking of doing that in a follow up anyway.